pkg/providers/imagefamily/azlinux.go

```diff
@@ -149,14 +150,16 @@ func (u AzureLinux) ScriptlessCustomData(kubeletConfig *bootstrap.KubeletConfigu
 }
 
 // UserData returns the default userdata script for the image Family
-func (u AzureLinux) CustomScriptsNodeBootstrapping(kubeletConfig *bootstrap.KubeletConfiguration,
+func (u AzureLinux) CustomScriptsNodeBootstrapping(
+	kubeletConfig *bootstrap.KubeletConfiguration,
 	taints []v1.Taint,
 	startupTaints []v1.Taint,
 	labels map[string]string,
 	instanceType *cloudprovider.InstanceType,
 	imageDistro string,
 	storageProfile string,
-	nodeBootstrappingClient types.NodeBootstrappingAPI) customscriptsbootstrap.Bootstrapper {
+	nodeBootstrappingClient types.NodeBootstrappingAPI,
+	fipsMode *v1beta1.FIPSMode) customscriptsbootstrap.Bootstrapper {
```

Comment:
very minor style thing:

Generally prefer the ends of these multiline method signatures to be:
```suggestion
	fipsMode *v1beta1.FIPSMode,
) customscriptsbootstrap.Bootstrapper {
```

The (minor) advantage of that is that when you add a new parameter the diff is cleaner because it doesn't look like you deleted/changed the return type (because it's on its own line)